package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@kazoohr/typeorm-pagination",
-  "repository": "github:KazooHR/typeorm-pagination",
+  "name": "@WorkTangohr/typeorm-pagination",
```

Comment:
Same nit [here](https://github.com/KazooHR/typeorm-pagination/pull/4/files#r1584858519) if you make that other change.